src/ui/toolbar/MainToolBar.qml

```diff
@@ -356,6 +356,7 @@ Rectangle {
             anchors.top:        parent.top
             anchors.bottom:     parent.bottom
             exclusiveGroup:     mainActionGroup
+            visible:            activeVehicle ? activeVehicle.supportsMissionManager || QGroundControl.forceShowMissionManager : true
```

Comment:
This ends up being a bit wierd since force setting is global, but in reality it's only for ArduSub. Problem down the road:
* New plugin turns off supportsMissionManager
* User turns on force mission for ArduSub, leaves it on but then switches away from ArduSub as offline selection.  Force setting is still on, but not visible and will affect all plugins which return supportsMissionManager == false